tests/unit-core/test-api-set-and-clear-error-flag.c

```diff
@@ -31,6 +31,82 @@ main (void)
 
   JERRY_ASSERT (obj_val != err_val);
   jerry_release_value (err_val);
+  jerry_release_value (obj_val);
+
+  jerry_value_t str = jerry_create_string ((jerry_char_t *) "Pterodactylus");
+  jerry_value_t error = jerry_create_error_from_value (str, true);
+  str = jerry_get_value_from_error (error, true);
+  jerry_size_t str_size = jerry_get_string_size (str);
+  jerry_char_t str_buff[str_size];
+  jerry_string_to_utf8_char_buffer (str, str_buff, str_size);
+  str_buff[str_size] = '\0';
```

Comment:
`str_buff[str_size] = '\0';` this is a memory overwrite since `jerry_char_t str_buff[str_size];` is defined this way. I would recommend to remove `str_buff[str_size] = '\0';` and use `memcmp()` instead of `strcmp()` below, since `memcmp()` has a size argument. Furthermore check the string size with a separate assert right after `jerry_get_string_size`.

You can also add a:
```
const char *pterodactylus_p = "Pterodactylus";
const size_t pterodactylus_size = strlen(pterodactylus_p);
```
at the beginning and use these values for assertion checks.